examples/calibrationDataRead/calibrationDataRead.ino

```diff
@@ -0,0 +1,57 @@
+/*
+ * This example shows how to read the TDS and EC calibration values
+ * from Amperka TDS-sensor
+ * During the calibration, use standart buffer solution of know TDS value
+ * For more details read the calibration section in API.md
+ */
+
+// Include the library to comunicate with Amperka TDS-sensor
+#include <AmperkaTDS.h>
+
+// Use any analog GPIO pin of the controller
+constexpr auto pinSensor = A0;
+
+// Create an object to work with Amperka TDS-sensor
+AmperkaTDS sensorTDS(pinSensor);
+
+void setup() {
+    // Open serial port with 9600 baud rate
+    Serial.begin(9600);
+    // Wait for serial port to connect, needed for native USB.
+    while (!Serial)
+      ;
+    // Start the sensor for communication
+    sensorTDS.begin();
+    // Use this function only for calibration sensor,
+    // otherwise comment-out.
```

Comment:
This is an example of a sensor calibration. No need to comment on the calibration function.